.findbugs/findbugs-exclude.xml

```diff
@@ -16,5 +16,8 @@
   <Match>
     <Class name="~io\.fabric8\.kubernetes\.api\.model\.(Doneable).+(\$.*)?" />
   </Match>
+  <Match>
+    <Class name="~io\.strimzi\.test\.Extensions\.StrimziExtension.*" />
```

Comment:
Any reason to exclude StrimziExtension for findbugs? I guess we need to care about the code in this class and we need to use findbugs to prevent some errors.

Reply:
Feel free to try remove this one and fix the issues. I wasn't able to achieve that before weekend.

Reply:
After fixing package names I don't have any problems with findbugs, so you can remove this excluding.